### PR TITLE
209 compute b allele fraction files for variant calling

### PR DESCRIPTION
**Pull request by holtgrewe**

### CODE REVIEW

File: snappy_pipeline/workflows/variant_calling/__init__.py [hunk 1]
```diff
@@ -183,6 +183,9 @@
 # Default configuration variant_calling
 step_config:
   variant_calling:
+    baf_file_generation:
+      min_ad: 10
```
Comment (eudesbarbosa) [inline snappy_pipeline/workflows/variant_calling/__init__.py:187]: What is this argument and where are you using it?

File: snappy_pipeline/workflows/variant_calling/__init__.py [hunk 2]
```diff
@@ -1022,6 +1025,76 @@ def get_resource_usage(self, action):
...
+        """Return output files that all germline variant calling sub steps must return (VCF +
+        TBI file)
+        """
+        # Validate action
+        self._validate_action(action)
+        ext_names = {"bw": ".bw", "bw_md5": ".bw.md5"}
+        for key, ext in ext_names.items():
+            yield key, self.base_path_out + ext
+
+    def get_log_file(self, action):
```
Comment (eudesbarbosa) [inline snappy_pipeline/workflows/variant_calling/__init__.py:1073]: Perhaps also log conda information? (requires changes to wrapper as well).

**Review by eudesbarbosa — COMMENTED** [review]
Looks good, left two questions...

File: snappy_pipeline/workflows/variant_calling/__init__.py (hunk 1 above)
Comment (holtgrewe) [reply snappy_pipeline/workflows/variant_calling/__init__.py:187]: Good point, adding note and actually using it.

File: snappy_pipeline/workflows/variant_calling/__init__.py (hunk 2 above)
Comment (holtgrewe) [reply snappy_pipeline/workflows/variant_calling/__init__.py:1074]: done

**Review by eudesbarbosa — APPROVED** [review]
LGTM

**Pull request merged**